tools/diffscripts/README.md

```diff
@@ -0,0 +1,208 @@
+# CSharp SDK Diff Automation
+
+## Overview
+During the release process we need to examine the changes to the SDK surface to make sure we're not making too many changes, breaking or otherwise. This tool will automate the process of generating the mark down files so we do not need to use the GUI version which is time consuming.
+
+
+## Quickstart
+This quickstart assumes that you clone your repositories to `c:\repos` and that you've already cloned the CSharp SDK to `c:\repos\azure-iot-sdk-csharp`
```

Comment:
Are you just saying the example steps use a repo root of "c:\repos", so change that out with whatever repo root you prefer?

Or are you saying there is an explicit assumption in another script that you've cloned to that root?

Your statement sounds like the latter, but I'm hoping you mean the former.

Reply:
The script makes no assumption about the locaiton of the SDK direcroty and is all relative. Meaning you could have your repos in  `c:\mylocalroot\anotherdirectory\csharp` ... HOWEVER it **will** look for `c:\mylocalroot\anotherdirectory\iot-sdks-internals` you would want to override this is with the `-SDKInternalsPath` and the script will tell you to do so.